test/extended/builds/cluster_config.go

```diff
@@ -107,12 +202,26 @@ var _ = g.Describe("[Feature:Builds][Serial][Slow][Disruptive] alter builds via
 
 		g.Context("build config context", func() {
 
+			g.AfterEach(func() {
```

Comment:
this replaces the aforementioned non-functional apply to reset the config ... I also moved it within the scope of just the build tests